
Just a dirty hack to build on current Debian sid.  Won't go upstream.

--- grub2-1.95+20080128/include/grub/powerpc/libgcc.h   2008-01-10 07:27:08.000000000 +0000
+++ buildd/grub2-1.95+20080128/include/grub/powerpc/libgcc.h    2008-01-30 22:39:43.000000000 +0000
@@ -25,21 +25,14 @@
 void EXPORT_FUNC (__fixunsdfsi) (void);
 void EXPORT_FUNC (__floatsidf) (void);
 void EXPORT_FUNC (__floatsisf) (void);
+void EXPORT_FUNC (__floatundisf) (void);
 void EXPORT_FUNC (__lshrdi3) (void);
-void EXPORT_FUNC (__make_dp) (void);
-void EXPORT_FUNC (__make_fp) (void);
 void EXPORT_FUNC (__muldf3) (void);
 void EXPORT_FUNC (__mulsf3) (void);
-void EXPORT_FUNC (__pack_d) (void);
-void EXPORT_FUNC (__pack_f) (void);
 void EXPORT_FUNC (__subdf3) (void);
 void EXPORT_FUNC (__subsf3) (void);
-void EXPORT_FUNC (__thenan_df) (void);
-void EXPORT_FUNC (__thenan_sf) (void);
 void EXPORT_FUNC (__trampoline_setup) (void);
 void EXPORT_FUNC (__truncdfsf2) (void);
 void EXPORT_FUNC (__ucmpdi2) (void);
-void EXPORT_FUNC (__unpack_d) (void);
-void EXPORT_FUNC (__unpack_f) (void);
 void EXPORT_FUNC (__floatdisf) (void);
 void EXPORT_FUNC (__cmpdi2) (void);
